### PR TITLE
build: add truncateLength as parameter

### PR DESCRIPTION
**Pull request by Kikobeats**

### CODE REVIEW

File: packages/metascraper-description/README.md [hunk 1]
```diff
@@ -11,6 +11,19 @@
...
+## API
+
+### description([options])
+
+#### options
+
+##### truncateLength
+
+Type: `string`<br>
+Default: `300`
```
Comment (daviswhitehead) [inline packages/metascraper-description/README.md]: should this be a string type or number?

File: packages/metascraper-helpers/index.js [hunk 2]
```diff
@@ -165,9 +165,13 @@ const extension = (str = '') => {
...
-const description = value => isString(value) && getDescription(value)
+const description = (value, opts) =>
+  isString(value) && getDescription(value, opts)
 
-const getDescription = (str, opts) => {
+const getDescription = (
+  str,
+  { truncateLength = TRUNCATE_MAX_LENGTH, ...opts } = {}
+) => {
   const description = removeLocation(truncate(str, TRUNCATE_MAX_LENGTH))
```
Comment (daviswhitehead) [inline packages/metascraper-helpers/index.js]: do you need to change this line at all?

File: packages/metascraper-description/README.md (hunk 1 above)
Comment (daviswhitehead) [reply packages/metascraper-description/README.md]: If this is the way to add the option, then wouldn't it be a number?
const metascraper = require('metascraper')([
  require('metascraper-description')({
    truncateLength: 500
  })
])

Comment (Kikobeats) [reply packages/metascraper-description/README.md]: yes!

File: packages/metascraper-helpers/index.js (hunk 2 above)
Comment (Kikobeats) [reply packages/metascraper-helpers/index.js]: true

**Pull request merged**